doc/source/whatsnew/v1.4.2.rst

```diff
@@ -23,7 +23,7 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
--
+- Fixed "longtable" formatting in :meth:`.Styler.to_latex` when ``column_format`` is given in extended format (:issue:`46037`)
```

Comment:
Was this working in 1.3.x or a new feature in 1.4?

Reply:
longtable was made available in 1.4

Reply:
Thanks. Could you merge in main one more time?

Reply:
done and green